src/ClientSymbols/QSettingsBasedStorageManager.cpp

```diff
@@ -70,4 +74,28 @@ QSettingsBasedStorageManager::LoadModuleSymbolFileMappings() {
   return mappings;
 }
 
+void QSettingsBasedStorageManager::SaveDisabledModulePaths(absl::flat_hash_set<std::string> paths) {
+  settings_.beginWriteArray(kDisabledModulesKey, static_cast<int>(paths.size()));
+  int index = 0;
+  for (const auto& path : paths) {
+    settings_.setArrayIndex(index);
+    settings_.setValue(kDisabledModuleKey, QString::fromStdString(path));
+    index++;
+  }
+  settings_.endArray();
+}
+
+[[nodiscard]] absl::flat_hash_set<std::string>
+QSettingsBasedStorageManager::LoadDisabledModulePaths() {
+  const int size = settings_.beginReadArray(kDisabledModulesKey);
+  absl::flat_hash_set<std::string> paths{};
```

Comment:
I checked the google style guide and couldn't find anything about it. So I guess its up to preference. I do like `absl::flat_hash_set<std::string> paths{};` because I feel it signals explicitly that there are no arguments.. but without the curly braces there is nothing confusing.. I haven't fully made up my mind yet

Reply:
I am going to merge without changing it back, because I do not want to wait for the CI again

Reply:
[TotW#88](https://g3doc.corp.google.com/devtools/library_club/g3doc/totw/88.md?cl=head)
> Use the traditional constructor syntax (with parentheses) when the initialization is performing some active logic, rather than simply composing values together.

Does hash table initialization count as "active logic"?